config/importer/gajim.go

```diff
@@ -296,13 +296,13 @@ func mergeAccountInformation(ac gajimAccountInfo, s gajimOTRSettings, s2 map[str
 		}
 	}
 
+	res.Proxies = make([]string, 0)
 	if ac.proxy != "" {
-		res.Proxies = []string{ac.proxy}
-		res.RequireTor = true
+		res.Proxies = append(res.Proxies, ac.proxy)
 	}
 
 	if strings.HasSuffix(res.Server, ".onion") {
-		res.RequireTor = true
+		res.Proxies = append(res.Proxies, "tor-auto://")
```

Comment:
yeah, we probably should just ignore this case, otherwise it may have conflict with other proxy.